lib/generation/prototyping/pb_prototype_linker_service.dart

```diff
@@ -33,6 +33,9 @@ class PBPrototypeLinkerService {
 
     while (stack.isNotEmpty) {
       var currentNode = stack.removeLast();
+      if (currentNode == null) {
+        continue;
```

Comment:
Might be worthwhile exploring why the `attributeNodes` contain a null element 🤔

Reply:
This occurs for `inherited_scaffold.dart`

What happens is that when we initialize the scaffold we give it the attribute `body`. When we initialize it, it is null until a value is given to the `attributeNode` of the `body` attribute. 

A fix for this would be to set the `body` attribute only when we want to already set it with a value, rather than when we initialize the Scaffold.